src/QuickMarcEditor/QuickMarcEditor.test.js

```diff
@@ -322,6 +325,594 @@ describe('Given QuickMarcEditor', () => {
         expect(getByText('ui-quick-marc.authority-record.edit.title')).toBeDefined();
       });
     });
+
+    describe('when 010 $a field is updated', () => {
+      describe('when click on save&close button', () => {
+        it('should open confirmation modal', () => {
+          const { getByTestId, getByText } = renderQuickMarcEditor({
+            marcType: MARC_TYPES.AUTHORITY,
+            numOfLinks: 1,
+            initialValues: {
+              leader: 'assdfgs ds sdg',
+              records: [
+                {
+                  tag: 'LDR',
+                  content: 'assdfgs ds sdg',
+                  id: 'LDR',
+                },
+                {
+                  _isDeleted: false,
+                  _isLinked: false,
+                  content: '$a n  931006645',
+                  id: 'af322498-097b-4c74-af15-c9166884aa6a',
+                  indicators: ['\\', '\\'],
+                  isProtected: false,
+                  tag: '010',
+                },
+                {
+                  _isDeleted: false,
+                  _isLinked: false,
+                  content: '$a Yuan, Bing',
+                  id: 'e95af4e5-008c-42c4-999c-4e103da9de13',
+                  indicators: ['1', '\\'],
+                  isProtected: false,
+                  tag: '100',
+                },
+              ],
+            },
+          });
+
+          const contentField = getByTestId('content-field-1');
+
+          fireEvent.change(contentField, { target: { value: '$a n  9310066' } });
+          fireEvent.click(getByText('stripes-acq-components.FormFooter.save'));
+
+          expect(getByText('Confirmation modal')).toBeDefined();
+        });
+
+        it('should close the modal, save the updates and close the editor on clickng save and close button', async () => {
+          const { queryByText, getByTestId, getByText } = renderQuickMarcEditor({
+            marcType: MARC_TYPES.AUTHORITY,
+            numOfLinks: 1,
+            initialValues: {
+              leader: 'assdfgs ds sdg',
+              records: [
+                {
+                  tag: 'LDR',
+                  content: 'assdfgs ds sdg',
+                  id: 'LDR',
+                },
+                {
+                  _isDeleted: false,
+                  _isLinked: false,
+                  content: '$a n  931006645',
+                  id: 'af322498-097b-4c74-af15-c9166884aa6a',
+                  indicators: ['\\', '\\'],
+                  isProtected: false,
+                  tag: '010',
+                },
+                {
+                  _isDeleted: false,
+                  _isLinked: false,
+                  content: '$a Yuan, Bing',
+                  id: 'e95af4e5-008c-42c4-999c-4e103da9de13',
+                  indicators: ['1', '\\'],
+                  isProtected: false,
+                  tag: '100',
+                },
+              ],
+            },
+          });
```

Comment:
It duplicates lines 332-363, I suggest moving this to `quickMarcEditorProps` constant and reuse